### PR TITLE
Cls2 89 add to list header

### PR DESCRIPTION
**Pull request by CaitBarnard**
## Description of change

Move 'add to list' to a chip in the header and remove the button from the dropdown menu

_Document what the PR does and why the change is needed_

## Test instructions

_What should I see?_

### Before

Screenshot 2023-05-05 at 10 55 58

### After

Screenshot 2023-05-05 at 11 03 51

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/client/components/CompanyLocalHeader/index.jsx [hunk 1]
```diff
@@ -145,23 +174,22 @@ const CompanyLocalHeader = ({
                 </Button>
               </StyledButtonContainer>
 
-              <ConnectedDropdownMenu
-                label="View options"
-                closedLabel="Hide options"
-                id="local_header"
-                dataTest="local-header-options-dropdown"
+              <StyledSecondaryButton
```
Comment (chopkinsmade) [inline src/client/components/CompanyLocalHeader/index.jsx:177]: This button doesn't do anything anymore when clicked, it should redirect to the export page when clicked. The test in the deleted options-spec.js file should have tested the click not just the link url, can you update the tests to try clicking both of these links and asserting the url is correct

File: src/client/components/Chip/index.jsx
Comment (chopkinsmade) [inline src/client/components/Chip/index.jsx:1]: Are the changes made to this file needed? In the src/client/components/CompanyLocalHeader/index.jsx file this component isn't being used, the styling is being added to a `chip` html component, not this React component

File: src/client/components/CompanyLocalHeader/index.jsx [hunk 2]
```diff
@@ -6,17 +6,22 @@ import Main from '@govuk-react/main'
 import GridCol from '@govuk-react/grid-col'
 import GridRow from '@govuk-react/grid-row'
 import Button from '@govuk-react/button'
+// import SecondaryButton from '../SecondaryButton'
```
Comment (chopkinsmade) [inline src/client/components/CompanyLocalHeader/index.jsx:9]: Some commented out code in this file

File: src/client/components/CompanyLocalHeader/index.jsx [hunk 3]
```diff
@@ -40,11 +45,33 @@ const StyledButtonContainer = styled('div')`
   display: inline-block;
 `
 
+const StyledList = styled('div')`
+  padding-bottom: 10px;
+`
+
+const StyledChip = styled('chip')`
```
Comment (chopkinsmade) [inline src/client/components/CompanyLocalHeader/index.jsx:52]: As mentioned in slack, <chip> is not a valid HTML DOM element so is causing this warning in the console view. Maybe try using a button instead?
image

File: test/functional/cypress/specs/companies/local-header/buttons-spec.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    it('should display the add export project link', () => {
+      assertExportProjectButton(exportProjectUrl)
+    })
+
+    it('should click the add export project link', () => {
+      cy.get('[data-test=header-add-export-project]').click()
+      cy.location().should((loc) => {
+        expect(loc.pathname + loc.search).to.eq(exportProjectUrl)
+      })
+      cy.go('back')
```
Comment (chopkinsmade) [inline test/functional/cypress/specs/companies/local-header/buttons-spec.js:27]: In your `beforeEach` you're visiting the page every time so you don't need the `cy.go('back')`.

File: test/functional/cypress/support/company-local-header-assertions.js [hunk 5]
```diff
@@ -28,11 +28,23 @@ const assertBadgeText = (badgeText) => {
...
+ * Asserts that the add to list chip has the correct URL
  */
-const assertButtons = (url) => {
-  cy.contains('View options').click()
-  cy.contains('Add to or remove from lists').should('have.attr', 'href', url)
+const assertChip = (addRemoveFromListUrl, detailsUrl) => {
+  cy.get('[data-test="add-to-list-chip"]').contains('+ Add to list')
+  cy.get('[data-test="add-to-list-chip"]').click()
+  cy.location().should((loc) => {
+    expect(loc.pathname).to.eq(`${addRemoveFromListUrl}`)
```
Comment (chopkinsmade) [inline test/functional/cypress/support/company-local-header-assertions.js:37]: You don't need string interpolation here, you can use `expect(loc.pathname).to.eq(addRemoveFromListUrl)`

File: test/functional/cypress/support/company-local-header-assertions.js [hunk 6]
```diff
@@ -28,11 +28,23 @@ const assertBadgeText = (badgeText) => {
...
+    expect(loc.search).contains(`${detailsUrl}`)
```
Comment (chopkinsmade) [inline test/functional/cypress/support/company-local-header-assertions.js:38]: Same as above, can get rid of the string interpolation

**Review by chopkinsmade — APPROVED** [review]

**Review by backslashbaker — APPROVED** [review]

**Pull request merged**